### ISSUE TITLE
Torchmetric to pytorch-forecasting for classification tasks

### ISSUE THREAD

**Issue by Kaszanas** [jdb78/pytorch-forecasting#1227]
### Expected behavior

I would like to know a little more about what methods are required for the ```TemporalFusionTransformer``` model metric to be working correctly with this library. The documentation mentions a couple of models that should work for classification tasks yet it is very vague on how to accomplish this.

### Actual behavior

When passing 

### Code to reproduce the problem

The full code to reproduce the problem is not feasible to be copied here.

Conversion function for torchmetric -> forecasting does not seem to work correctly.

```python
from torchmetrics.classification import Accuracy

logging_metrics = nn.ModuleList(
    [convert_torchmetric_to_pytorch_forecasting_metric(Accuracy)()]
)
```

Therefore I guess that it is required to implement the Accuracy metric in some specific way for it to work with a ```TemporalFusionTransformer```.
